In [ ]:
import time
cand77 = 0
cand77 = 0
cand39 = 0
cand66 = 0
nulo = 0
voto = 0
branco = 0
while(voto != 444444):
  voto = int(input("Digite seu voto"))
  match(voto):
    case 77:
      cand77 +=1
      print('Você votou no candidato 77')
    case 39:
     cand39 +=1
     print('Você votou no candidato 39')
    case 66:
      cand66+=1
      print('Você votou no candidato 66')
    case 0:
      nulo +=1
      print('Você votou nulo')
    case _:
      branco +=1
      print('Você votou em branco')
  print('\n')
  time.sleep(2)

votos_validos = cand77 + cand39 + cand66
print('\nVotos no candidato 77: ', cand77)
print('\nVotos no candidato 39: ', cand39)
print('\nVotos no candidato 66: ', cand66)
print('\nVotos em branco: ', branco)
print('\nVotos válidos: ', votos_validos)



Digite seu voto39
Você votou no candidato 39


Digite seu voto66
Você votou no candidato 66


Digite seu voto77
Você votou no candidato 77


Digite seu voto0
Você votou nulo


Digite seu voto45
Você votou em branco


Digite seu voto4444444
Você votou em branco




1 - Transcreva o algoritmo a seguir para Python:



```python
# Transcreva o algoritmo a seguir para Python:
ALGORITMO TABUADA_9;
VAR
N, X, R : INTEIRO;
INICIO
  PARA X DE 1 ATE 10 FACA
  INICIO
    R ← 9 * X;
    ESCREVER ( X, ‘ x 9 = ’, R);
  FIM;
FIM_PARA;
FIM.
```




FIM.

In [ ]:
for i in range(11):
  R = 9 * i;
  print(i, ' x 9 = ', R);


0  x 9 =  0
1  x 9 =  9
2  x 9 =  18
3  x 9 =  27
4  x 9 =  36
5  x 9 =  45
6  x 9 =  54
7  x 9 =  63
8  x 9 =  72
9  x 9 =  81
10  x 9 =  90


2 - Faça um programa em Python que mostre todos os números inteiros ímpares situados na faixa
de 1000 a 1500.

In [ ]:
for i in range(1000, 1500):
  if(i%2 == 1):
    print(i)



1001
1003
1005
1007
1009
1011
1013
1015
1017
1019
1021
1023
1025
1027
1029
1031
1033
1035
1037
1039
1041
1043
1045
1047
1049
1051
1053
1055
1057
1059
1061
1063
1065
1067
1069
1071
1073
1075
1077
1079
1081
1083
1085
1087
1089
1091
1093
1095
1097
1099
1101
1103
1105
1107
1109
1111
1113
1115
1117
1119
1121
1123
1125
1127
1129
1131
1133
1135
1137
1139
1141
1143
1145
1147
1149
1151
1153
1155
1157
1159
1161
1163
1165
1167
1169
1171
1173
1175
1177
1179
1181
1183
1185
1187
1189
1191
1193
1195
1197
1199
1201
1203
1205
1207
1209
1211
1213
1215
1217
1219
1221
1223
1225
1227
1229
1231
1233
1235
1237
1239
1241
1243
1245
1247
1249
1251
1253
1255
1257
1259
1261
1263
1265
1267
1269
1271
1273
1275
1277
1279
1281
1283
1285
1287
1289
1291
1293
1295
1297
1299
1301
1303
1305
1307
1309
1311
1313
1315
1317
1319
1321
1323
1325
1327
1329
1331
1333
1335
1337
1339
1341
1343
1345
1347
1349
1351
1353
1355
1357
1359
1361
1363
1365
1367
1369
1371
1373
1375
1377
1379
1381
1383
1385
1387
1389
1391
1393
1395
1397
1399


3. Elabore um programa em Python que calcule e escreva a soma de 10 números lidos.

In [ ]:
i = 0
soma = 0
while(i < 10):
  num = float(input('Insira o '+str(i+1)+' ° número'))
  i+= 1
  soma += num
print('\nA soma dos números inseridos é: ', soma)

Insira o 1 ° número1
Insira o 2 ° número2
Insira o 3 ° número3
Insira o 4 ° número4
Insira o 5 ° número5
Insira o 6 ° número6
Insira o 7 ° número7
Insira o 8 ° número8
Insira o 9 ° número9
Insira o 10 ° número10
A soma dos números inseridos é:  55.0


4 - Faça um programa em Python que leia a nota de 10 alunos de uma turma. Ao final, deve ser
escrita a média geral da turma.

In [ ]:
nota = 0
soma = 0
soma_notas = 0
i = 0
while(i < 10):
  nota = int(input('Insira a nota do '+str(i+1)+' ° aluno'))
  soma_notas += nota
  i+=1
MD = soma_notas/10
print('\nA média da turma foi: ', MD)

Insira a nota do 1 ° aluno5
Insira a nota do 2 ° aluno5
Insira a nota do 3 ° aluno5
Insira a nota do 4 ° aluno5
Insira a nota do 5 ° aluno5
Insira a nota do 6 ° aluno5
Insira a nota do 7 ° aluno5
Insira a nota do 8 ° aluno5
Insira a nota do 9 ° aluno5
Insira a nota do 10 ° aluno5

A média da turma foi:  5.0


5 - Faça um programa em Python que leia 5 números. Para cada número fornecido, escreva se
ele é NULO, NEGATIVO ou POSITIVO.

In [ ]:
i = 0
while(i < 5):
  num = int(input('\nInsira o '+str(i+1)+' ° número'))
  if(num > 0):
    print('O número ', num, ' é positivo')
  elif(num < 0):
    print('O número ', num, ' é negativo')
  else:
    print('O número ', num, ' é nulo')
  i+=1



Insira o 1 ° número1
O número  1  é positivo

Insira o 2 ° número2
O número  2  é positivo

Insira o 3 ° número3
O número  3  é positivo

Insira o 4 ° número4
O número  4  é positivo

Insira o 5 ° número5
O número  5  é positivo


6 - Ler dois valores inteiros e escrever os números em sequência do menor até maior valor.

In [ ]:
num1 = int(input('Insira um número'))
num2 = int(input('Insira outro número'))
if(num1 > num2):
  print('\n',num2, ' > ',num1)
elif(num2 > num1):
  print('\n',num1, ' > ',num2)

Insira um número2
Insira outro número1

 1  >  2


7 - Faça um programa em Python que escreva os números pares entre 100 e 200.

In [ ]:
for i in range(100, 201):
  if(i%2 == 0):
    print(i)

100
102
104
106
108
110
112
114
116
118
120
122
124
126
128
130
132
134
136
138
140
142
144
146
148
150
152
154
156
158
160
162
164
166
168
170
172
174
176
178
180
182
184
186
188
190
192
194
196
198
200


8 - Escrever um programa em Python que leia um valor para n, inteiro e positivo, e escreva a
tabuada de n. Com os respectivos valores.

In [ ]:
n = int(input('Insira um valor inteiro e positivo'))
for i in range (11):
  print(i, ' * ', n, ' = ', n*i)


Insira um valor inteiro e positivo5
0  *  5  =  0
1  *  5  =  5
2  *  5  =  10
3  *  5  =  15
4  *  5  =  20
5  *  5  =  25
6  *  5  =  30
7  *  5  =  35
8  *  5  =  40
9  *  5  =  45
10  *  5  =  50


9 - Escrever um programa em Python que gera os números de 1000 a 1999 e escreve aqueles
que divididos por 11 dão um resto igual a 5.

In [ ]:
for i in range (1000, 2000):
  if(i % 11 == 5):
    print(i)

1006
1017
1028
1039
1050
1061
1072
1083
1094
1105
1116
1127
1138
1149
1160
1171
1182
1193
1204
1215
1226
1237
1248
1259
1270
1281
1292
1303
1314
1325
1336
1347
1358
1369
1380
1391
1402
1413
1424
1435
1446
1457
1468
1479
1490
1501
1512
1523
1534
1545
1556
1567
1578
1589
1600
1611
1622
1633
1644
1655
1666
1677
1688
1699
1710
1721
1732
1743
1754
1765
1776
1787
1798
1809
1820
1831
1842
1853
1864
1875
1886
1897
1908
1919
1930
1941
1952
1963
1974
1985
1996


10 - Faça um programa em Python que leia um número qualquer (N), e realize a soma dos
números de 1 até N. Ao final, deve-se escrever o resultado.

In [ ]:
N = int(input('Insira um número'))
soma = 0
i = 0
for i in range (0, N+1):
  soma += i

print('A soma de 1 até ', N,' é igual a: ', soma)

Insira um número3
A soma de 1 até  3  é igual a:  6


11 - Faça um programa em Python para ler 10 valores inteiros. Para cada valor fornecido escreva
uma mensagem que indica se cada valor fornecido é PAR ou ÍMPAR.

In [ ]:
for i in range (0, 10):
  N = int(input('Insira o '+ str(i+1) +'° número'))
  if(N % 2 == 0):
    print('O número ', N, 'é par\n')
  else:
    print('O número ', N, 'é ímpar\n')

Insira o 1° número2
O número  2 é par

Insira o 2° número5
O número  5 é ímpar

Insira o 3° número6
O número  6 é par

Insira o 4° número7
O número  7 é ímpar



KeyboardInterrupt: Interrupted by user

12. Faça um programa em Python que leia a idade de 10 alunos, calcule e escreva a média de
idade destes.

In [ ]:
soma_idades = 0
alunos = 3
for i in range (0, alunos):
  idade = int(input('Insira a idade do '+ str(i+1) +'° aluno'))
  soma_idades += idade
MD = soma_idades / alunos
print(f'A média de idades é de: {MD:.0f}'+ ' anos')

KeyboardInterrupt: Interrupted by user

13 - Faça um algoritmo em Python que leia dois valores inteiros e escreva se eles estão em ordem
crescente, decrescente ou são iguais, após isso escreva todos os valores do menor ao maior.

In [ ]:
num1 = int(input('Insira um número'))
num2 = int(input('Insira outro número'))
if(num1 > num2):
  print('Os números estão em ordem decrescente')
  print('\n')
  print('Do menor para o maior: ', num2, ' ', num1)
elif(num2 > num1):
  print('Os números estão em ordem crescente')
  print('\n')
  print('Do menor para o maior: ', num1, ' ', num2)
else:
  print('Os números são iguais')



Insira um número1
Insira outro número2
Os números estão em ordem crescente


Do menor para o maior:  1   2


14. Escrever um programa em Python que lê 10 valores para n, um de cada vez, todos inteiros e
positivos, e para cada n lido, escreva a tabuada de 1 até n de n.

* 1 x n = n
* 2 x n = 2n
* ...
* n x n = n

In [ ]:
for k in range (10):
  n = int(input('Insira um número ('+ str(k)+')'))
  for i in range (0, 11):
    print(n, ' * ', i, ' = ', i*n)
  print('\n')

Insira um número (0)1
1  *  0  =  0
1  *  1  =  1
1  *  2  =  2
1  *  3  =  3
1  *  4  =  4
1  *  5  =  5
1  *  6  =  6
1  *  7  =  7
1  *  8  =  8
1  *  9  =  9
1  *  10  =  10




KeyboardInterrupt: Interrupted by user

1 - Escreva um programa que lê um número inteiro N do usuário. O programa deverá escrever na
tela o fatorial desse número (N!).
* Exemplo de entrada:
* 5
* Saída:
* 120
* Exemplo de entrada:
* 8
* Saída:
* 40320

In [ ]:
num = int(input('Insira um número'))
res = 1;

for i in range (1, num+1):
  res *= i
  print(i)
print(res)

Insira um número5
1
2
3
4
5
120


2 - Escreva um programa para uma urna eletrônica. O programa fica lendo os votos até que o
usuário informe o valor 4444444 (código de finalização da urna). Após a finalização da urna, o
programa deverá escrever na tela a quantidade de votos e o percentual de votos para cada uma
das opções abaixo:
a. Votos no Candidato 77
b. Votos no Candidato 39
c. Votos no Candidato 66
d. Voto em branco (0)
e. Votos nulos (nenhuma das opções anteriores)
f. Votos válidos (aqueles computados para algum candidato)
g. Votos não válidos

In [ ]:
import time
cand77 = 0
cand77 = 0
cand39 = 0
cand66 = 0
nulo = 0
voto = 0
branco = 0
while(voto != 4444444):
  voto = int(input("Digite seu voto"))
  match(voto):
    case 77:
      cand77 +=1
      print('Você votou no candidato 77')
    case 39:
     cand39 +=1
     print('Você votou no candidato 39')
    case 66:
      cand66+=1
      print('Você votou no candidato 66')
    case 0:
      branco +=1
      print('Você votou em branco')
    case 4444444:
      break
    case _:
      nulo +=1
      print('Você votou nulo')
  print('\n')
  time.sleep(2)

votos_validos = cand77 + cand39 + cand66
print('\n')
print('Votos no candidato 77: ', cand77)
print('Votos no candidato 39: ', cand39)
print('Votos no candidato 66: ', cand66)
print('Votos em branco: ', branco)
print('Votos válidos: ', votos_validos)



Digite seu voto0
Você votou em branco


Digite seu voto1
Você votou nulo


Digite seu voto66
Você votou no candidato 66


Digite seu voto4444444
39

Votos no candidato 77:  0
Votos no candidato 39:  0
Votos no candidato 66:  1
Votos em branco:  1
Votos válidos:  1
